components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -224,8 +224,9 @@ function CustomLeague:defineCustomPageVariables(args)
 
 	Variables.varDefine('game', GameLookup.getName({args.game}))
 	Variables.varDefine('tournament_game', GameLookup.getName({args.game}))
-	Variables.varDefine('tournament_patch', args.patch)
-	Variables.varDefine('patch', args.patch)
+	Variables.varDefine('tournament_patch', args.patch or args.voobly)
+	Variables.varDefine('patch', args.patch or args.voobly)
```

Comment:
is `voobly` an alias for patch or does it indicate something different?
might be good to add a comment here to explain it